.luacheckrc

```diff
@@ -13,11 +13,13 @@ files["awesomerc.lua"].allow_defined_top = true
 -- This file itself
 files[".luacheckrc"].ignore = {"111", "112", "131"}
 
+-- Theme files, ignore max line length
+files["themes/*"].ignore = {"631"}
```

Comment:
I've meant to explicitly ignore lines that are too long, but I think configuring it like above is better.